README.md

```diff
@@ -10,7 +10,10 @@ The config data provided here gets processed in the [veda-data-airflow](https://
 
 ![veda-data-publication][veda-data-publication]
 
-(See `/docs/publishing-data-annotated` for the technical details of how this automated process is being developed and implemented.)
+> [!NOTE] See these links for technical details about the automated publishing methods veda-data provides:
+>
```

Comment:
I don't know why the [!NOTE] callout formatting isn't working in the preview. Maybe remove the extra > line? Or are the bullets the problem?

Reply:
I think I fixed it!

Reply:
yay!